zonemaster/zonemaster-engine
# Update the translation to French (Engine)

## Purpose

Updates updated or added msgids into French msgstrs. And removes obsolete entries.
Mainly for DNSSEC.

## Context

Resolved zonemaster/zonemaster-engine#1004

## Changes

Update fr.po

## How to test this PR

Use The following command to check :
`zonemaster-cli --locale fr_FR.UTF-8 [Zone to test]`

List of zones used in unit tests to test against :
```
dnssec01-ds-algorithm-deprecated1.zut-root.rd.nic.fr
dnssec01-ds-algorithm-deprecated3.zut-root.rd.nic.fr
dnssec01-ds-algorithm-not-ds.zut-root.rd.nic.fr
dnssec01-ds-algorithm-ok.zut-root.rd.nic.fr
dnssec01-ds-algorithm-reserved.zut-root.rd.nic.fr
dnssec01-nxdomain.zut-root.rd.nic.fr
dnssec02-dnskey-ksk-not-sep.zut-root.rd.nic.fr
dnssec02-ds-does-not-match-dnskey.zut-root.rd.nic.fr
dnssec02-no-common-keytags.zut-root.rd.nic.fr
dnssec03-many-iterations.zut-root.rd.nic.fr
dnssec03-no-nsec3param.zut-root.rd.nic.fr
dnssec03-too-many-iterations.zut-root.rd.nic.fr
dnssec04-duration-long.zut-root.rd.nic.fr
dnssec04-remaining-long.zut-root.rd.nic.fr
dnssec05-algorithm-deprecated.zut-root.rd.nic.fr
dnssec05-algorithm-private.zut-root.rd.nic.fr
dnssec05-algorithm-reserved.zut-root.rd.nic.fr
dnssec05-algorithm-unassigned.zut-root.rd.nic.fr
dnssec06-extra-processing-broken-1.zut-root.rd.nic.fr
dnssec06-extra-processing-broken-2.zut-root.rd.nic.fr
dnssec07-dnskey-but-not-ds.zut-root.rd.nic.fr
dnssec07-ds-but-not-dnskey.zut-root.rd.nic.fr
dnssec07-neither-dnskey-nor-ds.zut-root.rd.nic.fr
dnssec08-dnskey-not-signed.zut-root.rd.nic.fr
dnssec08-dnskey-rrsig-not-yet-valid.zft-root.rd.nic.fr
dnssec08-dnskey-signature-not-ok-broken.zut-root.rd.nic.fr
dnssec08-dnskey-signature-not-ok.zut-root.rd.nic.fr
dnssec08-rrsig-no-matching-dnskey.zft-root.rd.nic.fr
dnssec09-rrsig-no-matching-dnskey.zft-root.rd.nic.fr
dnssec09-rrsig-not-valid-by-dnskey.zft-root.rd.nic.fr
dnssec09-soa-not-signed.zft-root.rd.nic.fr
dnssec09-soa-rrsig-expired.zft-root.rd.nic.fr
dnssec09-soa-rrsig-not-yet-valid.zft-root.rd.nic.fr
dnssec10-inconsistent-nsec-nsec3.zft-root.rd.nic.fr
dnssec10-non-existent-domain-name-exists-01.zft-root.rd.nic.fr
dnssec10-non-existent-domain-name-exists-02.zft-root.rd.nic.fr
dnssec10-non-existent-domain-name-exists-03.zft-root.rd.nic.fr
dnssec10-non-existent-domain-name-exists-04.zft-root.rd.nic.fr
dnssec10-non-existent-domain-name-exists-05.zft-root.rd.nic.fr
dnssec11-ds-but-unsigned.zft-root.rd.nic.fr
dnssec11-inconsistent-dnskey.zft-root.rd.nic.fr
dnssec11-inconsistent-ds.dnssec11-parent.zft-root.rd.nic.fr
dnssec13-algo-not-signed-dnskey.zft-root.rd.nic.fr
dnssec13-algo-not-signed-ns.zft-root.rd.nic.fr
dnssec13-algo-not-signed-soa.zft-root.rd.nic.fr
dnssec15-cdnskey-no-cds.zft-root.rd.nic.fr
dnssec15-cds-cdnskey-01.zft-root.rd.nic.fr
dnssec15-cds-cdnskey-02.zft-root.rd.nic.fr
dnssec15-cds-cdnskey-03.zft-root.rd.nic.fr
dnssec15-cds-no-cdnskey.zft-root.rd.nic.fr
dnssec15-inconsistent-cdnskey-01.zft-root.rd.nic.fr
dnssec15-inconsistent-cdnskey-02.zft-root.rd.nic.fr
dnssec15-inconsistent-cdnskey-03.zft-root.rd.nic.fr
dnssec15-inconsistent-cdnskey-04.zft-root.rd.nic.fr
dnssec15-inconsistent-cds-01.zft-root.rd.nic.fr
dnssec15-inconsistent-cds-02.zft-root.rd.nic.fr
dnssec15-inconsistent-cds-03.zft-root.rd.nic.fr
dnssec15-inconsistent-cds-04.zft-root.rd.nic.fr
dnssec15-mismatch-cds-cdnskey.zft-root.rd.nic.fr
dnssec15-no-cds-no-cdnskey.zft-root.rd.nic.fr
dnssec16-cds-cdnskey-invalid-rrsig-01.zft-root.rd.nic.fr
dnssec16-cds-cdnskey-invalid-rrsig-02.zft-root.rd.nic.fr
dnssec16-cds-cdnskey-signed.zft-root.rd.nic.fr
dnssec16-cds-cdnskey-unsigned.zft-root.rd.nic.fr
dnssec16-cds-matches-no-dnskey.zft-root.rd.nic.fr
dnssec16-cds-matches-non-sep-dnskey.zft-root.rd.nic.fr
dnssec16-cds-matches-non-zone-dnskey.zft-root.rd.nic.fr
dnssec16-cds-signed-by-unknown-dnskey.zft-root.rd.nic.fr
dnssec16-cds-unsigned.zft-root.rd.nic.fr
dnssec16-cds-without-dnskey.zft-root.rd.nic.fr
dnssec16-delete-cds.zft-root.rd.nic.fr
dnssec16-mixed-delete-cds.zft-root.rd.nic.fr
dnssec17-cdnskey-is-non-zone.zft-root.rd.nic.fr
dnssec17-cdnskey-matches-no-dnskey.zft-root.rd.nic.fr
dnssec17-cdnskey-signed-by-unknown-dnskey.zft-root.rd.nic.fr
dnssec17-cdnskey-unsigned.zft-root.rd.nic.fr
dnssec17-delete-cdnskey.zft-root.rd.nic.fr
dnssec17-mixed-delete-cdnskey.zft-root.rd.nic.fr
dnssec18-no-match-cdnskey-rrsig-ds.zft-root.rd.nic.fr
dnssec18-no-match-cds-rrsig-ds.zft-root.rd.nic.fr
```
